app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -167,6 +167,22 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
         vm.cloudVolumeModel.aws_volume_type !== 'standard');
   };
 
+  vm.awsBaseSnapshotChanged = function(baseSnapshotId) {
+    vm.encryptionDisabled = false;
+    $timeout(function() {
+      if (baseSnapshotId) {
+        for (var i = 0; i < vm.baseSnapshotChoices.length; i++) {
+          if (vm.baseSnapshotChoices[i].ems_ref === baseSnapshotId) {
+            vm.cloudVolumeModel.aws_encryption = vm.baseSnapshotChoices[i].encrypted;
```

Comment:
@sasoc I just encountered a case where the `Base snapshot` `encrypted` state was `undefined`. So when that happens the bootstrap switch for Encryption looks like this - 
<img width="388" alt="screen shot 2017-07-07 at 3 25 13 pm" src="https://user-images.githubusercontent.com/1538216/27979123-839bdb22-6328-11e7-9314-6bcbccd919f3.png">

In order to avoid that, can we change the above line to -
```javascript
vm.cloudVolumeModel.aws_encryption = angular.isDefined(vm.baseSnapshotChoices[i].encrypted) && vm.baseSnapshotChoices[i].encrypted ? true : false;
```